api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -58,16 +60,14 @@ public UserService(Provider<User> userProvider,
       Clock clock,
       Random random,
       FireCloudService fireCloudService,
-      Provider<WorkbenchConfig> configProvider,
-      Provider<ComplianceService> complianceServiceProvider) {
+      Provider<WorkbenchConfig> configProvider) {
```

Comment:
I was actually suggesting:

```
public UserService(
    ...
    ComplianceService complianceService) {
  ...
  this.complianceService = complianceService;
```

YOu don't need the provider in this case because ComplianceService is a singleton.